apps/researcher/src/messages/en/messages.json

```diff
@@ -200,7 +200,7 @@
     "orderByCount": "Order by count",
     "selectedFilters": "Selected filters",
     "aboutFacetsHeader": "About these facets",
-    "aboutFacetsText": "The data of these facet items are collected from multiple datasets. We are in the process of improving the quality of these facets items."
+    "aboutFacetsText": "The data of these facet items are collected from multiple datasets. We are in the process of improving the quality of these facets items.\n\nPlease note that currently, for some facets the values for search filters are provided in Dutch, regardless of the selected interface language.  We're working on adding translations soon."
```

Comment:
Change `...some facets the values...` into `...some facets the items...`, to be consistent with the use of `facet items` in the first and second sentences?